runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/grpcclient/GrpcWindmillServer.java

```diff
@@ -360,69 +343,34 @@ private <ResponseT> ResponseT callWithBackoff(Supplier<ResponseT> function) {
 
   @Override
   public GetWorkResponse getWork(GetWorkRequest request) {
-    if (syncApplianceStub == null) {
-      return callWithBackoff(
-          () ->
-              syncStub()
-                  .withDeadlineAfter(unaryDeadlineSeconds, TimeUnit.SECONDS)
-                  .getWork(
-                      request
-                          .toBuilder()
-                          .setJobId(options.getJobId())
-                          .setProjectId(options.getProject())
-                          .setWorkerId(options.getWorkerId())
-                          .build()));
-    } else {
-      return callWithBackoff(
-          () ->
-              syncApplianceStub
-                  .withDeadlineAfter(unaryDeadlineSeconds, TimeUnit.SECONDS)
-                  .getWork(request));
+    if (syncApplianceStub != null) {
+      return callWithBackoff(() -> syncApplianceStub.getWork(request));
     }
+
+    throw new RpcException(
+        new UnsupportedOperationException(
+            "Unary GetWork calls are not supported with Windmill service."));
```

Comment:
Streaming Engine is what dataflow docs refer to it as, can you update here and below? I see some usage of this text predates your change but let's make it clearer

Reply:
done.

Reply:
done.